ui/hooks/useAccountTotalFiatBalance.js

```diff
@@ -54,10 +54,11 @@ export const useAccountTotalFiatBalance = (
   const primaryTokenImage = useSelector(getNativeCurrencyImage);
   const nativeCurrency = useSelector(getNativeCurrency);
 
-  const { loading, tokensWithBalances } = useTokenTracker({
+  const loading = false;
+  const { tokensWithBalances } = useTokenTracker({
+    chainId: currentChainId,
     tokens,
-    address: account?.address,
-    includeFailedTokens: true,
+    address: account.address,
```

Comment:
Do we still want to `useTokenTracker` here? My understanding was that it is shimmed for backwards compatibility. Can't we now read the balances directly from state?

Reply:
I think the change you see here is the one coming from brian's PR, i dnt think i updated this piece;
But i think that eventually yes we would move away from this hook,